types/levelup/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for levelup 4.3
+// Type definitions for levelup 5.1
```

Comment:
The types aren't identical with 5.x.x Please see the release documentation
https://github.com/Level/levelup/releases

1. You need to remove the default export.
2. add new method `db.getMany(keys)`
3. add new read only property `supports.status`
4. and something more https://github.com/Level/levelup/commit/e1ecad9

Reply:
As far as I can see `db.getMany(keys)` is the only one missing - but in either case I will make that change in a separate PR. Thanks!

Reply:
Actually I will continue to bring the types up to date with the current version of the library - it makes more sense rather than working around the issue.